src/main/java/com/minecolonies/coremod/blocks/BlockHutMiner.java

```diff
@@ -1,5 +1,7 @@
 package com.minecolonies.coremod.blocks;
 
+import net.minecraft.block.Block;
+import net.minecraftforge.registries.IForgeRegistry;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraftforge.registries.IForgeRegistry'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)